TipCatDotNet.Api/Services/HospitalityFacilities/IMemberService.cs

```diff
@@ -14,9 +14,9 @@ public interface IMemberService
 
         Task<Result<MemberResponse>> AddCurrent(string? identityClaim, CancellationToken cancellationToken = default);
 
-        Task<Result<MemberResponse>> RegenerateQR(MemberContext memberContext, int memberId, int accountId, CancellationToken cancellationToken = default);
+        Task<Result<MemberResponse>> RegenerateQR(MemberContext memberContext, int memberId, int acccountId, CancellationToken cancellationToken = default);
 
-        Task<Result<List<MemberResponse>>> Get(MemberContext memberContext, int accountId, CancellationToken cancellationToken = default);
+        Task<Result<List<MemberResponse>>> GetAll(MemberContext memberContext, int accountId, CancellationToken cancellationToken = default);
```

Comment:
Signatures of Get All and Get One are different. In such cases better to keep the similar name for both methods